### ISSUE TITLE
a11y: Color Contrast - Cluster Dashboard

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#13733]
Insufficient color contrast for standard text

## Actual result

The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.

## Instances

"Foreground Text Color: #3D98D3 
Background Text Color: #FFFFFF
Ratio: 3.16:1 
 - Main Content
  - Links such as Cluster Tools, Events, Certificates, Pod coredns-576bfc4dc7-2pwl7, coredns-576bfc4dc7-2pwl7.1802f7e9644bd8a4, Full events list, Full secrets list, serving-cert, k3s-serving and so on (in default state)

Foreground Text Color: #3D98D3 
Background Text Color: #F4F5FA
Ratio: 2.9:1
 - Main Content
  - Links such as Pod coredns-576bfc4dc7-2pwl7, coredns-576bfc4dc7-2pwl7.1802f7e9644bd8a4, serving-cert, k3s-serving and so on (on hover and in selected state)

Foreground Text Color: #B6B6C2 
Background Text Color: #FFFFFF
Ratio: 2.00:1
 - Main Content
  - 6 / 110
  - 0.1 / 4 cores
  - 0.07 / 7.75 GiB
  - 0 / 7.75 GiB"

## Expected results

The contrast ratio between foreground text and background for the standard text should be sufficient.

**Comment by rak-phillip** [comment]
Fixed in VPAT

Issue 120
